kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/Fetcher.java

```diff
@@ -30,10 +30,11 @@
     /**
      * Open a stream of messages, starting at the position indicated by the given {@code token}.
      *
-     * @param token the token representing positions of the partitions to start from
+     * @param token   the token representing positions of the partitions to start from
+     * @param groupId a {@link String} defining the Consumer Group id the fetcher should start Consumer instances in
      * @return a {@link BlockingStream} providing messages from Kafka
      */
-    BlockingStream<TrackedEventMessage<?>> start(KafkaTrackingToken token);
+    BlockingStream<TrackedEventMessage<?>> start(KafkaTrackingToken token, String groupId);
```

Comment:
The similar approach like I proposed earlier.

Reply:
Definitely a valid argument, but I think breaking changes are justified as long as we do not have a full release of the extension. 
As soon as we have a 4.0 version of the extension, I'd fully agree with this point.

Reply:
Discussed with @m1l4n54v1c and approved.